experimenter/experimenter/features/management/commands/load_feature_configs.py

```diff
@@ -33,9 +61,38 @@ def handle(self, *args, **options):
                 },
             )
 
+            feature_configs[(feature.application_slug, feature.slug)] = feature_config
+
             if not created:
                 features_to_disable.discard((feature.application_slug, feature.slug))
 
+        # Disable any features that we didn't come across.
+        for application_slug, feature_slug in features_to_disable:
+            if feature_slug not in NO_FEATURE_SLUG:
+                logger.info(f"Feature Not Found in YAML: {feature_slug}")
+                feature_config = NimbusFeatureConfig.objects.filter(
+                    application=application_slug, slug=feature_slug
+                ).update(enabled=False)
+
+        # Populate the version table and cache the results so we can re-use them
+        # in model creation.
+        versions: dict[Version, NimbusFeatureVersion] = {}
+        for feature in Features.versioned():
+            if feature.version in versions:
+                continue
+
+            versions[feature.version] = NimbusFeatureVersion.objects.get_or_create(
+                major=feature.version.major,
+                minor=feature.version.minor,
+                patch=feature.version.patch,
+            )[0]
```

Comment:
Oh becuase it returns the tuple of `(instance, created_bool)`?  I always hate that XD  Another way to write this is

```py
versions[feature.version], _ = NimbusFeatureVersion.objects.get_or_create(.....)
```

instead of tacking `[0]` at the end.  Not really much different but makes it clear it's a 2tuple that we only care about the first field, whereas `[0]` could be read as 'there's some arbitrary list here and we're just pulling the first instance out'.  But doesn't really matter either way.

Reply:
I'm personally not a fan of destructing with `_` in Python because its an actual variable and not some placeholder. Plus there is a common pattern in Django of:

```
from django.utils.translation import gettext as _
```

which conflicts with using _. I know we don't support i10n, but maybe one day we do and this would be a footgun.

Reply:
Yeah I know, if you use an actual variable there the linter would complain it wasn't being used but knows to ignore `_`.  I wish there was a better pattern for this in Python.